code/EditableSpamProtectionField.php

```diff
@@ -263,7 +259,15 @@ public function getRequired()
 
     public function getIcon()
     {
-        return 'spamprotection/images/' . strtolower($this->class) . '.png';
+        // Get the end of the full qualified class name
+        $shortClass = end(explode("\\", __CLASS__));
+
+        $resource = ModuleLoader::getModule('silverstripe/spamprotection')
+            ->getResource('images/' . strtolower($shortClass) . '.png');
```

Comment:
Not the shorthand? Does it not work for Images?

Reply:
If you mean the `vendor/module:resource` syntax - it does work for images yeah

Reply:
Yeah that's the one. Just curious as to why you didn't use it here.

Reply:
Ah - as far as I know we need to return the resolved URL rather than the reference to it. I did just notice that we're missing this update in userforms too, so will make a PR for that now too :)

Reply:
https://github.com/silverstripe/silverstripe-userforms/pull/683

Reply:
I'll make a follow up PR for this module as well

Reply:
https://github.com/silverstripe/silverstripe-spamprotection/pull/60